README.md

```diff
@@ -1,20 +1,20 @@
-### OpenCV: Open Source Computer Vision Library
+## OpenCV: Open Source Computer Vision Library
 
-#### Resources
+### Resources
 
 * Homepage: <http://opencv.org>
 * Docs: <http://docs.opencv.org/master/>
 * Q&A forum: <http://answers.opencv.org>
 * Issue tracking: <https://github.com/opencv/opencv/issues>
```

Comment:
Links are clickable(at least in my browser) so IMHO it's ok to retain them as is.